### PR TITLE
Tidy and update docs

### PR DESCRIPTION
**Pull request by tom-parker**
@commuterjoy @wheresrhys @adambraimbridge

### CODE REVIEW

File: server/lib/get-allocation-id.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ module.exports = function(req) {
 		.then(function(response) {
 			if (response.status >= 400) {
 				throw new Error('Invalid session');
+				metrics.count('id.invalid-session-token', 1);
```
Comment (adambraimbridge) [inline server/lib/get-allocation-id.js]: :)

File: server/lib/get-allocation-id.js [hunk 2]
```diff
@@ -55,13 +56,8 @@ module.exports = function(req) {
 
 	// If neither ft-session-token nor allocation ID were provided,
 	// use a random-generated uuid.
-	if (isAnonymous) {
```
Comment (commuterjoy) [inline server/lib/get-allocation-id.js]: intentional?

**Comment by commuterjoy** [comment]
+1 aside from comment

File: server/lib/get-allocation-id.js (hunk 2 above)
Comment (tom-parker) [inline server/lib/get-allocation-id.js]: Yes, this was just unnecessary logic I think. All other cases are covered above so if we get this far we need to return a new UUID whatever

Comment (commuterjoy) [inline server/lib/get-allocation-id.js]: cool. ship it.

**Pull request merged**